Traceback (most recent call last):
  File "main.py", line 98, in <module>
    main()
  File "main.py", line 15, in main
    cf = config.Load()
  File "/home/grupo03/M5/Code/config/configuration.py", line 26, in Load
    cf = yaml.load(f, Loader=yaml.Loader)
  File "/home/grupo03/.local/lib/python2.7/site-packages/ruamel/yaml/main.py", line 935, in load
    return loader._constructor.get_single_data()
  File "/home/grupo03/.local/lib/python2.7/site-packages/ruamel/yaml/constructor.py", line 109, in get_single_data
    node = self.composer.get_single_node()
  File "/home/grupo03/.local/lib/python2.7/site-packages/ruamel/yaml/composer.py", line 78, in get_single_node
    document = self.compose_document()
  File "/home/grupo03/.local/lib/python2.7/site-packages/ruamel/yaml/composer.py", line 101, in compose_document
    node = self.compose_node(None, None)
  File "/home/grupo03/.local/lib/python2.7/site-packages/ruamel/yaml/composer.py", line 138, in compose_node
    node = self.compose_mapping_node(anchor)
  File "/home/grupo03/.local/lib/python2.7/site-packages/ruamel/yaml/composer.py", line 211, in compose_mapping_node
    while not self.parser.check_event(MappingEndEvent):
  File "/home/grupo03/.local/lib/python2.7/site-packages/ruamel/yaml/parser.py", line 141, in check_event
    self.current_event = self.state()
  File "/home/grupo03/.local/lib/python2.7/site-packages/ruamel/yaml/parser.py", line 578, in parse_block_mapping_key
    token.start_mark,
ruamel.yaml.parser.ParserError: while parsing a block mapping
  in "config/classification_bts.yml", line 2, column 1
expected <block end>, but found u'<scalar>'
  in "config/classification_bts.yml", line 15, column 32
